### PR TITLE
Added rawData in CreateHeader

### PR DESCRIPTION
**Pull request by MingboPeng**
Add or modify raw data's header directly in CreateHeader component

### CODE REVIEW

File: src/Ladybug_Create LB Header.py [hunk 1]
```diff
@@ -92,13 +101,24 @@ def setDefaults():
     
     return checkData, location, dataType, units, timeStep, analysisPeriod
 
+def checkRawData(rawD):
+    lb_preparation = sc.sticky["ladybug_Preparation"]()
+    indexList, oldHeader = lb_preparation.separateList(rawD, lb_preparation.strToBeFound)
+    separatedLists = rawD[indexList[0]+7:indexList[1]]
```
Comment (mostaphaRoudsari) [inline src/Ladybug_Create LB Header.py]: @MingboPeng what will happen if rawD doesn't have a header?

Comment (MingboPeng) [inline src/Ladybug_Create LB Header.py]: @mostaphaRoudsari  it will add a default header that CreateHeader originally generates

Comment (mostaphaRoudsari) [inline src/Ladybug_Create LB Header.py]: @MingboPeng right! thanks. I should have checked the `separateList` method. It should work fine.

**Comment by chriswmackey** [comment]
@MingboPeng ,
I forgot to tell you that this is very useful.  I've already used this input a few times in this last month.

**Pull request merged**